Files_external_setup_via_command_line.md

```diff
@@ -0,0 +1,93 @@
+ | Remove an applicable group to a DropBox mountpoint via command line | The mountpoint is configured and applicable group condition is removed | :construction:  |1 | Remove an applicable group to a GoogleDrive mountpoint via command line | The mountpoint is configured and applicable group condition is removed | :construction:  |
+ | Remove an applicable group to a SFTP mountpoint via command line | The mountpoint is configured and applicable group condition is removed | :construction:  |
+ | Remove all applicable conditions of a S3 mountpoint via command line | The mountpoint is configured and applicable conditions are removed | :construction:  |
+ | Remove all applicable conditions of a DropBox mountpoint via command line | The mountpoint is configured and applicable conditions are removed | :construction:  |
+ | Remove all applicable conditions of a GoogleDrive mountpoint via command line | The mountpoint is configured and applicable conditions are removed | :construction:  |
+ | Remove all applicable conditions of a SFTP mountpoint via command line | The mountpoint is configured and applicable conditions are removed | :construction:  |
+| **files_external:backends** |  |   |
+ |  |  |   |
+ | After configuring some external storages moutnpoints, run backends command via CLI | A list of available authentication and their config options is shown | :construction:  |
+ | **files_external:config** |  |   |
+ |  |  |   |
+ | Configure a S3 mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a DropBox mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a FTP mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a GoogleDrive mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a Local mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a OpenStack mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a ownCloud mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a SFTP mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a SMB mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | Configure a webDAV mountpoint via command line | The mountpoint is configured with the new data and appears in the server webUI | :construction:  |
+ | **files_external:create** |  |   |
```

Comment:
Create should be the first command to use. It is needed to have storages mounted for testing the rest.